crates/rspack_plugin_javascript/src/visitors/inject_runtime_helper.rs

```diff
@@ -81,6 +81,9 @@ impl<'a> VisitMut for InjectRuntimeHelper<'a> {
         return;
       }
 
+      if word == "_instanceof" {
```

Comment:
What's this for? Would you mind adding some comments?

Reply:
![image](https://user-images.githubusercontent.com/17974631/233820303-ea8e60a1-f186-4592-96da-6754ea2eaa42.png)
I think this is enough.

Reply:
let me add some comments.

Reply:
Done